### DISCUSSION TITLE
How to set the nav title and header-title separately.

### DISCUSSION THREAD

**Discussion by michaelcrook** [squidfunk/mkdocs-material#2565] [Q&A] [answered]
I am trying to change the nav title to "Table of Contents", currently it uses the site_name from the mkdocs.yml file so I am looking to separate the nav title from the header-title/site_name while making it easy to localize in the future. Any suggestions on the best way to do this?

**Comment by squidfunk** [comment, marked as answer]
The easiest way is to override `nav.html` (see the customization guide) and set your title there.

**Reply by EdouardThiel** [reply]
You can also see a solution in #2550.
